pipeline.py

```diff
@@ -91,7 +96,41 @@ def validate_dataset(path_to_validate):
     """
     Validates folder with assets
     """
-    pass
+    path = pathlib.Path(path_to_validate)
+
+    if not path_to_validate.exists():
+        raise FileNotFoundError
+
+    if not path.is_dir():
+        raise NotADirectoryError
+
+    if not path.glob('*'):
+        raise EmptyDirectoryError
+
+    files = {
+        "json": 0,
+        "pdf": 0,
+        "txt": 0
+    }
+    for file in path.iterdir():
+        file_name = file.name
+        if not re.match(r'\d+_(raw\.(txt|pdf)|meta\.json)', file_name):
+            raise InconsistentDatasetError
+
+        file_number = int(re.findall(r'\d+', file_name)[0])
+        current_file_number = files.get(file.suffix)
+        if file_number - current_file_number != 1:
+            raise InconsistentDatasetError
+        files[file.suffix] = file_number
+
+    with open(CRAWLER_CONFIG_PATH) as file:
```

Comment:
why do you open config?

Reply:
I wanted to check the needed amount of files via `CRAWLER_CONFIG_PATH` as there should be a precise number of articles we expected to collect